package.json

```diff
@@ -37,7 +37,7 @@
     "csv-parser": "^3.0.0",
     "path": "^0.12.7",
     "toml": "^3.0.0",
-    "xrpl": "^4.2.0"
+    "xrpl": "^4.2.5"
```

Comment:
should remove `^` and pin the version so future compromised patches don't get included unintentionally